api4/user.go

```diff
@@ -84,6 +85,67 @@ func getUser(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getUsers(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
Yeah, the function can be written better and use a better pattern for the different query parameters. There is one combination in this for not_in_channel and team_id together. I've made a ticket for myself to revisit this